tutor/test_cc1_01_RecruitingTeachers.py

```diff
@@ -83,7 +84,8 @@ def test_admin_recruitment_and_promo_website_is_available_7751(self):
         self.ps.test_updates['passed'] = False
 
         # Test steps and verification assertions
-        raise NotImplementedError(inspect.currentframe().f_code.co_name)
+        self.teacher.driver.get('http://cc.openstax.org/')
```

Comment:
Use the internal functions for the users:
`self.teacher.get(<url>)`
`self.teacher.find(<by>, <string>)`
`self.teacher.find_all(<by>, <string>)`
That way we can change the underlying code in Staxing but not the test case files.